src/main/kotlin/org/jitsi/nlj/transform/node/outgoing/ProbingDataSender.kt

```diff
@@ -63,6 +63,25 @@ class ProbingDataSender(
     private var numProbingBytesSentRtx: Int = 0
     private var numProbingBytesSentDummyData: Int = 0
 
+    init {
+        streamInformationStore.onRtpPayloadTypesChanged { currentRtpPayloadTypes ->
+            if (currentRtpPayloadTypes.isEmpty()) {
+                videoPayloadTypes.clear()
+                rtxSupported = false
+            } else {
+                currentRtpPayloadTypes.values.forEach { pt ->
+                    if (!rtxSupported && pt is RtxPayloadType) {
```

Comment:
i do the `!rtxSupported` check to avoid logging adding the support multiple times, but i suppose it's also slightly more efficient :)